test/vstest.console.UnitTests/ExecutorUnitTests.cs

```diff
@@ -338,6 +338,25 @@ public void ExecutorShouldPrintRunnerArchitecture()
         Assert.IsFalse(mockOutput.Messages.Any(message => message.Message!.Contains("vstest.console.exe is running in emulated mode")));
     }
 
+    [TestMethod]
+    [DataRow("--ShowDeprecateDotnetVSTestMessage")]
```

Comment:
I think you should use `DotNet`.

```suggestion
    [DataRow("--ShowDeprecateDotNetVSTestMessage")]
```

Reply:
Here `Dotnet` is the muxer `dotnet`, interesting...I've always used dotnet as single word

Reply:
What makes you think that? The product name is dotnet or dotNET, with the first one being used way more often imo. So Dotnet is correct. DotNET would also be correct, but dotNet is discouraged.

This guideline says to note the capitalization (and not spell with a space in the middle). 

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/net

This brand guide does not mention the spelling we are using `dotnet` explicitly, but uses it throughout the guide: https://github.com/dotnet/brand

Reply:
https://github.com/dotnet/brand/issues/26 added issue for that.